bom/pom.xml

```diff
@@ -470,6 +472,18 @@
             <artifactId>c3p0</artifactId>
             <version>${version.c3p0}</version>
          </dependency>
+
+         <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <version>${version.hikaricp}</version>
+         </dependency>
+
+         <dependency>
+            <groupId>org.jboss.slf4j</groupId>
+            <artifactId>slf4j-jboss-logging</artifactId>
```

Comment:
I'm not entirely sure if this should go to `bom`. Maybe `parent` would be a better candidate? 

//cc @tristantarrant